suricata.yaml.in

```diff
@@ -50,6 +50,7 @@ vars:
     GENEVE_PORTS: 6081
     VXLAN_PORTS: 4789
     TEREDO_PORTS: 3544
+    SIP_PORTS: "[5060, 5061]"
```

Comment:
This is meant to be used in signatures, right ?

`git grep SIP_PORTS` does not yield anything else

Reply:
yes, that's right.